app/app.go

```diff
@@ -43,6 +44,8 @@ type App struct {
 	Metrics          einterfaces.MetricsInterface
 	Saml             einterfaces.SamlInterface
 
+	diagnosticClient *analytics.Client
```

Comment:
This is a server wide feature so it should probably be on the Server struct and initialized and shutdown with the rest of the stuff there.

The app only lives for each request.

Reply:
I literally just figured that out in a quick chat with @jwilander. 

Thanks for confirming!